db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/HashableJoinStrategy.java

```diff
@@ -167,7 +167,8 @@ public boolean feasible(Optimizable innerTable,
         if (hashKeyColumns == null && skipKeyCheck) {
             // For full outer join, broadcast join is the default join strategy that can be applied
             // to any kinds of join predicate, so make the eligibility check more general
-            if (outerCost.getJoinType() == JoinNode.FULLOUTERJOIN &&
+            if ((outerCost.getJoinType() == JoinNode.FULLOUTERJOIN ||
+                 outerCost.getJoinType() == JoinNode.LEFTOUTERJOIN) &&
```

Comment:
@msirek should we also allow RIGHTOUTERJOIN here, or at this point we only have LEFT OUTER ?

Reply:
@dgomezferro For join algorithms we currently only have left outer join.  If we specify a right outer join in SQL it is executed using a left outer join algorithm (but with left and right tables swapped).  There is a test for it in BroadCastJoinIT.

Reply:
@msirek got it, thanks!